homeworks/nikolay.krasniuk_Kingstar22/7-oop/src/index.ts

```diff
@@ -0,0 +1,66 @@
+
+import './style.css';
+
+const uploadInputEl: HTMLElement = document.querySelector('[data-upload]');
+const displayContentEl: HTMLElement  = document.querySelector('[data-display-image]');
+const loaderEl: HTMLElement = document.querySelector('[data-loader]');
+const imageFocusEl: HTMLElement = document.querySelector('[data-enlarged-image]');
+
+class UploadImage {
+    constructor (
+        private uploadInputEl: HTMLElement,
+        private displayContentEl: HTMLElement,
+        private loaderEl: HTMLElement,
+        private imageFocusEl: HTMLElement,
+    ){
+        this.listenEvents();
+    }
+
+    private listenEvents() {
+        this.uploadInputEl.addEventListener('change', this.settingsHandler);
+        this.displayContentEl.addEventListener('click', (event) => {
+            this.loaderEl.classList.add('active');
+            setTimeout(() => {
+                this.showPreviewImages(event);
+            }, 500);
+        });
+        this.imageFocusEl.addEventListener('click', this.hideShowPreviewImages);
+    }
+
+    private settingsHandler = (event: any) => {
+        const files = event.target.files[0];
```

Comment:
Is it files? It seems like it's an individual file. Also, just destruct the first file from the new `files` variable mentioned below.